### PR TITLE
Attack

### PR DESCRIPTION
**Pull request by jwarshaw**
Create attack option in duel story and add link

### CODE REVIEW

**Comment by morosebose** [comment]
@casey-collab

**Comment by shersom** [comment]
Bad luck? @casey-collab

File: english/explore-outside/journey-to-flaming-tower/jellybus/ask-person-about-destination/ask-person-about-mission/duel/duel.md [hunk 1]
```diff
@@ -6,4 +6,8 @@ you in the least.
 to stop me and I'll prove that I'm the most
 powerful warrior in Chocosnackland!", you shout.
 
-He instantly leaps to his feet and lets out a roar.
\ No newline at end of file
+He instantly leaps to his feet and lets out a roar.
+
+Do you? 
+
+[Attack!](../attack/attack.md)
```
Comment (casey-collab) [inline english/explore-outside/journey-to-flaming-tower/jellybus/ask-person-about-destination/ask-person-about-mission/duel/duel.md]: It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.

File: english/explore-outside/journey-to-flaming-tower/jellybus/ask-person-about-destination/ask-person-about-mission/duel/attack/attack.md [hunk 2]
```diff
@@ -0,0 +1,8 @@
+You charge the roaring KitKat. His huge, wafer body sits atop two thin pretzel legs. He is top heavy. You decide to
```
Comment (casey-collab) [inline english/explore-outside/journey-to-flaming-tower/jellybus/ask-person-about-destination/ask-person-about-mission/duel/attack/attack.md]: It looks like this page is not reachable from the rest of the story. That is, if a reader starts in `language.md` and keeps clicking links, they will never find this page. You can fix this by finding a good place in the existing story to add an option with a link that leads to this page.

**Comment by casey-collab** [comment]
Besides my other comments, this looks good!  Once you respond to those, this should be good to merge.

**Comment by jwarshaw** [comment]
I have fixed the broken link thus making my page reachable from the story.

**Comment by shersom** [comment]
Not that you'll ever need to do this again, but you have to mention @casey-collab for it to work.

**Comment by casey-collab** [comment]
Looks great! :dart:

**Comment by jwarshaw** [comment]
Thank you!

**Comment by casey-collab** [comment]
Unfortunately, this pull request conflicts with another change. Could you merge the master branch of the original repository into your branch, resolve the merge conflicts, and update the pull request?

**Comment by casey-collab** [comment]
Besides my other comments, this looks good!  Once you respond to those, this should be good to merge.

**Pull request merged**